### DISCUSSION TITLE
Priority Handling of Drivers vs Svc vs Fw

### DISCUSSION THREAD

**Discussion by satzguyz** [nasa/fprime#1396] [Q&A]
Hello

How the priority of the Svc, Drivers and Fw are managed in the Fprime SW. Is there any means to control it

**Comment by timcanham** [comment]
Can you elaborate? Do you mean task priority? If so, priority is an argument to the active component instantiation, for example [here](https://github.com/nasa/fprime/blob/devel/Ref/Top/instances.fpp#L34). The FPP specification that discusses the priority argument is [here](https://fprime-community.github.io/fpp/fpp-users-guide.html#Defining-Component-Instances_Component-Instance-Definitions_Active-Components).
